.travis.yml

```diff
@@ -12,10 +12,6 @@ matrix:
 
     - python: "2.7"
       env: TOXENV=py27
-    - python: "3.4"
-      env: TOXENV=py34
-    - python: "3.4"
-      env: TOXENV=py34-no-typing
```

Comment:
You said that removing Python 3.4 fixes an issue with PyYAML.  Why does it do that?  I would rather keep testing 3.4 for now since it is supported by e.g. Pipenv.

Reply:
The problem is that `PyYAML 5.3` is explicitly not compatible with Python 3.4 then you have [this](https://travis-ci.org/theskumar/python-dotenv/jobs/636988224?utm_medium=notification&utm_source=github_status) on Travis. This is not related to my PR, and I would be happy to adopt another option.

Reply:
OK I understand.  Let's keep your commit then.  I'll see if it can be tested again, but later.